web/crux/crux-docker-entrypoint.sh

```diff
@@ -5,7 +5,7 @@ case $1 in
         exec npm run start:prod
     ;;
     "migrate")
-        exec npx prisma migrate deploy
+        exec npm run encrypt:migrate
```

Comment:
Encrypting and migrating are two different tasks. Do we really want to combine them, it would be clearer to perform them as two separate steps, IMHO.

Reply:
That's a totally fair point; however, if we already had secrets in the db they would have been lost after applying the migration.

Another possible solution would be to check on every start if we have remaining temp tables with secrets to be migrated, and process them before the startup.

Which one do you prefer?